src/libra_py/workflows/nbra/generate_data.py

```diff
@@ -247,63 +247,66 @@ def distribute_jobs(params):
     Returns:
         None
     """
-    critical_params = ['guess_input_template', 'trajectory_xyz_file', 'reference_input_template', 'istep', 
```

Comment:
can we please keep "istep" and "fstep"? these may be used in many places, so I'm not sure if the current change would be backward-compatible

Reply:
also, it seems that you replace input of two integers with a more complex input of the entire list